ctrl_frequency: 25.0
hardwareCtrl_frequency: 25.0
robot_spec:
    mass: 4.0
    link_length: 0.35
    actuator_number: 5
    dubins_radius: 0.7
    turning_radius: 0.5
    max_vel: 0.3
    min_vel: -0.3
    average_vel: 0.05 #0.35 0.3 #
    max_turningRate: 1.0
    min_turningRate: -1.0
    max_jointTorque: 5.0
    min_jointTorque: -5.0
    max_thrustForce: 3.0
    min_thrustForce: -3.0
    max_screwRPM: 150.0
    min_screwRPM: -150.0
    max_jointAngle: 1.0 # number in Dynamixel
    min_jointAngle: -1.0
low_pass_filter:
    Vel: 5.0
    Joint: 3.0
kinematic_controller:
    eta: 0.5
    k_et: 1.0 #2.0 #3.0 #10.0 #2.7
    k_en: 2.0 #1.5 #3.0 #10.0
    k_theta: 3.0 #1.0 #2.4 #2.4 #10.0 #15.0 #13.0
    k_s: 1.3
    theta_a: 1.5 #1.5 #1.5 #3.5
hydrodyanmics:
    damping: [-2.0, -2.3, -0.05]
    quadratic_damping: [-14.0, -25.0, -0.2]
dynamic_controller:
    k_1: [2.5, 2.5, 2.5]
    k_2: [0.2, 0.2, 0.2]
    k_3: [0.4, 0.4, 0.000]
    k_4: [0.2, 0.2, 0.000]
    lambda: 9
    observer_z: 0.01
